dessia_common/workflow/utils.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu July 7 14:52:32 2022
+
+@author: xavier-ferry
+"""
+
+from typing import List, Dict
+
+
+class ToScriptElement:
+    """
+    Class meant to improve to_script readability
+    """
+    before_declaration: str = None
+    declaration: str = None
+    imports_as_is: List[str] = None
+    imports: List[str] = None
+
+    def __init__(self, declaration: str, before_declaration: str = None,
+                 imports: List[str] = None, imports_as_is: List[str] = None):
+        self.before_declaration = before_declaration
+        self.declaration = declaration
+        self.imports = imports
+        self.imports_as_is = imports_as_is
+
+    def imports_to_str(self) -> str:
+        script_imports = ""
+        for module, class_list in self.get_import_dict().items():
+            script_imports += f"from {module} import {', '.join(class_list)}\n"
+
+        for import_as_is in self.imports_as_is:
+            import_str = f"import {import_as_is}\n"
+            if import_str not in script_imports:
+                script_imports += import_str
+
+        return script_imports
+
+    def get_import_dict(self) -> Dict[str, List[str]]:
+        imports_dict: Dict[str, List[str]] = {}
+        for c in self.imports:
+            module = '.'.join(c.split('.')[:-1])
+            class_ = c.split('.')[-1]
+            if imports_dict.get(module) is None:
```

Comment:
It might be more pythonic and clear to do it this way : 

`if module in imports_dict:`